<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayerWidget</class>
 <widget class="QWidget" name="MediaPlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="playlistView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>321</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="mediaDisplayView">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>721</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="controllerView">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>570</y>
     <width>721</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
